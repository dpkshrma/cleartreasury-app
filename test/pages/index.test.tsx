import React from "react";
import { render } from "../testUtils";
import Dashboard from "../../src/pages/index";

describe("Dashboard page", () => {
  it("matches snapshot", () => {
    const obj = { user: { username: "Test" } };
    const { asFragment } = render(<Dashboard user={obj} />, {});
    expect(asFragment()).toMatchSnapshot();
  });
});
