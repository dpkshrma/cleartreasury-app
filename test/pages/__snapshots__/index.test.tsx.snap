// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard page matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="m-12"
  >
    <h1
      class="text-4xl"
    >
      Welcome
    </h1>
    <p
      class="text-lg text-gray-500"
    >
      Where would you like to start?
    </p>
  </div>
</DocumentFragment>
`;
